src/chapters/radius-auth.xml

```diff
@@ -0,0 +1,251 @@
+            </varlistentry>
+            <varlistentry>
+                 <term><property>radius-key-type</property></term>
+                 <listitem>
+                     <para>The type of the key file specified by the <property>radius-key-file</property> 
+                        parameter.  If not specified, this defaults to pkcs12, the default used by 
+                        the JRadius library.</para>
+                 </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-key-password</property></term>
+                <listitem>
+                    <para>The password of the private key specified in the 
+                        <property>radius-key-file</property> parameter.  By default the extension   
+                        will not use any password when trying to open the key file.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-ca-file</property></term>
+                <listitem>
+                    <para>The absolute path to the file that stores the certificate authority
+                        certificates for encrypted connections to the RADIUS server.  By default
+                        a file with the name ca.crt in the GUACAMOLE_HOME directory will be used.
+                        </para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-ca-type</property></term>
+                <listitem>
+                    <para>The type of file store used for the certificate authority.  If not
+                        specified this defaults to pem.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-ca-password</property></term>
+                <listitem>
+                    <para>The password used to protect the certificate authority store, if
+                        any.  If unspecified the extension will attempt to read the CA
+                        store without any password..</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-trust-all</property></term>
+                <listitem>
+                    <para>A boolean value indicating whether or not the RADIUS extension
+                    should trust all certificates or verify them against known good
+                    certificate authorities.  By default this is false, and certificates
+                    will validated.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-retries</property></term>
+                <listitem>
+                    <para>The number of times the client will retry the connection to the 
+                        RADIUS server and not receive a response before giving up.  The default
```

Comment:
For clarity, rather than simply state the effective default value, we should also state the default behavior. This is done nicely for `radius-trust-all` above:

> By default this is false, and certificates will validated.

but here is less complete:

> The default is 5.

Something like "By default, the client will retry the connection at most 5 times." (or similar) would satisfy this need.

Reply:
Fixed.